portfolio/src/main/java/com/google/sps/utils/CommentDataStore.java

```diff
@@ -0,0 +1,37 @@
+package com.google.sps.utils;
+
+import com.google.appengine.api.datastore.*;
+import com.google.sps.data.Comment;
+import com.google.sps.data.EntityConvertible;
+import com.google.sps.data.EntityConvertibleCreator;
+
+/** Provides the service to interact with Datastore to store or load comment. */
+public class CommentDataStore {
+
+    static class Constants {
```

Comment:
Actually, these can simply be private static final String variables inside the parent class if it's not used anywhere else.

Reply:
Fixed